.github/workflows/scheduled_release.yml

```
@@ -0,0 +1,137 @@
+
+                    git tag -a "$NEW_TAG" -m "Release $NEW_TAG"
+                    git push origin "$NEW_TAG"
+                  else
+                    echo "Tag $NEW_TAG already exists, skipping."
+                    exit 1
+                  fi
+                  echo "new_tag=$NEW_TAG" >> $GITHUB_OUTPUT
+                  echo "previous_tag=$PREVIOUS_TAG" >> $GITHUB_OUTPUT
+
+            - name: Output tag information
+              if: ${{ steps.check_commits.outputs.has-changes == 'true' }}
```

actionlint (1.7.12)

[error] property "check_commits" is not defined in object type {tag_version: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)